pkg/generated/openapi/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
@nikhiljindal Can you comment on the federation types? I am not sure if we have a separate code generation for federation. If we do, you can separate them in a another PR (or I can do it if you teach me speak federation :) )